test/init/generate-path/init-generate-path.test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const rimraf = require('rimraf');
+const { join, resolve } = require('path');
+const { run } = require('../../utils/test-utils');
+
+const firstPrompt = 'Will your application have multiple bundles?';
+
+const genPath = join(__dirname, 'test-assets');
+
+describe('init generate-path flag', () => {
+    beforeAll(() => {
+        rimraf.sync(genPath);
+    });
+
+    it('should scaffold in the specified path with --generate-path', () => {
+        const { stdout } = run(__dirname, ['init', '--generate-path', 'test-assets', '--auto'], false);
```

Comment:
Can we refactor it to `webpack init ./path/to/something`? And why we need `--auto`?

Reply:
`init` already takes in a package (`webpack-scaffold-*`) as an arg. Being a subcommand having a flag to accept any other arguments would be better.

Reply:
I see

/cc @webpack/cli-team What do you think about `--generate-path` name for flag?

Reply:
Supplying `--auto` flag would allow us to go with the defaults.

Reply:
> What do you think about --generate-path name for flag?

what out `--dir`?

Reply:
@jamesgeorge007 What do you think about arguments name?

Reply:
IMO `--generate-path` sounds good.

Reply:
/cc @webpack/cli-team need more options

Reply:
Any issues with `--path` only?

Reply:
I think `--path` can be some misleading

Reply:
hmm, then `--generation-path` or `--generate-path` sounds good.

Reply:
`--generation-path` sounds good for me

Reply:
I can think of a shortened version: `--gen-path`. How does that sound?

Reply:
I prefer do not shorted, it can be difficult to read